lib/src/ruby/simple_resolution.cc

```diff
@@ -63,16 +63,24 @@ namespace facter { namespace ruby {
         }
 
         // Otherwise, we were given a command so execute it
-        auto result = execute(command_shell,
-            {command_args, expand_command(_command)},
-            option_set<execution_options> {
-                execution_options::defaults,
-                execution_options::redirect_stderr
+        bool success;
+        string output, error;
+        tie(success, output, error) = execute(
+            command_shell,
+            {
+                command_args,
+                expand_command(_command)
+            },
+            0,  // REVIEW: should setcode commands have a default timeout?
```

Comment:
Same as my comment about external facts above; I think we could add a global execution timeout on the command line, if anything.  May revisit in the future, but for now will remove the REVIEW comments.